chapters/VK_KHR_swapchain/wsi.txt

```diff
@@ -971,6 +971,23 @@ pname:timeout provided:
   * ename:VK_SUBOPTIMAL_KHR is returned if an image became available, and
     the swapchain no longer matches the surface properties exactly, but can:
     still be used to present to the surface successfully.
+ifdef::VK_FOOL_printed_surface[]
+  * ename:VK_ERROR_BAD_LUCK_FOOL is returned if the surface was created from
+    fname:vkCreatePrintedSurfaceFOOL and the printer no longer wishes to
+    cooperate.
+  * ename:VK_ERROR_FILTER_FAILED_FOOL is returned if the surface was created
+    from fname:vkCreatePrintedSurfaceFOOL and the printer driver has
+    encountered a fatal issue.
+  * ename:VK_ERROR_OUT_OF_INK_FOOL is returned if the surface was created
+    from fname:vkCreatePrintedSurfaceFOOL and the printer has run out of ink.
+  * ename:VK_ERROR_OUT_OF_PAPER_FOOL is returned if the surface was created
+    from fname:vkCreatePrintedSurfaceFOOL and the printer has run out of paper.
+  * ename:VK_ERROR_OUT_OF_TONER_FOOL is returned if the surface was created
+    from fname:vkCreatePrintedSurfaceFOOL and the printer has run out of toner.
+  * ename:VK_ERROR_PAPER_JAM_FOOL is returned if the surface was created
+    from fname:vkCreatePrintedSurfaceFOOL and the printer has encountered a
+    paper jam.
```

Comment:
`VK_ERROR_DEVICE_LOST` ought to be returnable also, in case somebody has "borrowed" the printer and forgotten to return it.